CHANGELOG.md

```diff
@@ -11,6 +11,13 @@
 * Adds launch.json option to suppress Just-In-Time compiler optimizations.
 * Due to the previous two items and work from the .NET Team, it is now possible to easily debug into ASP.NET itself in projects running against .NET Core 2.1 preview 1. Support for debugging into all the managed code in .NET Core will come in future .NET Core 2.1 builds. Instructions are in the [wiki](https://github.com/OmniSharp/omnisharp-vscode/wiki/Debugging-into-the-.NET-Framework-itself).
 
+#### Alternative Omnisharp Support
+
+* Adds support to use the "omnisharp.path" option to download alternative copies of OmniSharp. The possible values for this option are:
```

Comment:
Maybe describe the feature a little more?

"Enables the use of pre-release builds of Omnisharp. Downloading a pre-release build of Omnisharp allows the C# extension for VS Code to use features that have been merged into the "master" branch of omnisharp-roslyn [link] but that have not bee officially released"